_includes/pubbliche-amministrazioni.html

```diff
@@ -83,6 +83,29 @@ <h5 class="card-title text-primary mt-4">
       </div>
     </div>
   </div>
+  <div class="section section-firma section--wpointers">
```

Comment:
TIP: put this code in a specific "include file" to use here (as made in the homepage)